internal/services/paloalto/next_generation_firewall_vnet_local_rulestack_resource_test.go

```diff
@@ -0,0 +1,393 @@
+  rule_stack_id       = azurerm_palo_alto_local_rule_stack.test.id
+
+  network_profile {
+    public_ip_ids = [azurerm_public_ip.test.id]
+
+    vnet_configuration {
+      virtual_network_id  = azurerm_virtual_network.test.id
+      trusted_subnet_id   = azurerm_subnet.test1.id
+      untrusted_subnet_id = azurerm_subnet.test2.id
+    }
+  }
+}
+`, r.template(data), data.RandomInteger)
+}
+
+func (r NextGenerationFirewallVnetResource) requiresImport(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+
+%[1]s
+
+resource "azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack" "import" {
+  name                = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.name
+  resource_group_name = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.resource_group_name
+  rule_stack_id       = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.rule_stack_id
+
+  network_profile {
+    public_ip_ids = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.network_profile.0.public_ip_ids
+
+    vnet_configuration {
+      virtual_network_id  = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.network_profile.0.vnet_configuration.0.virtual_network_id
+      trusted_subnet_id   = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.network_profile.0.vnet_configuration.0.trusted_subnet_id
+      untrusted_subnet_id = azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack.test.network_profile.0.vnet_configuration.0.untrusted_subnet_id
+    }
+  }
+}
+`, r.basic(data))
+}
+
+func (r NextGenerationFirewallVnetResource) complete(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+%[1]s
+
+resource "azurerm_public_ip" "egress" {
+  name                = "acctestpublicip-%[2]d-e"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  allocation_method   = "Static"
+  sku                 = "Standard"
+}
+
+
+resource "azurerm_palo_alto_next_generation_firewall_vnet_local_rulestack" "test" {
+  name                = "acctest-ngfwvn-%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  rule_stack_id       = azurerm_palo_alto_local_rule_stack.test.id
+
+  network_profile {
+    public_ip_ids     = [azurerm_public_ip.test.id]
+    egress_nat_ip_ids = [azurerm_public_ip.egress.id]
+
+    vnet_configuration {
+      virtual_network_id  = azurerm_virtual_network.test.id
+      trusted_subnet_id   = azurerm_subnet.test1.id
+      untrusted_subnet_id = azurerm_subnet.test2.id
+    }
+  }
+
+  dns_settings {
+    dns_servers = ["8.8.8.8", "8.8.4.4"]
+  }
+
+  destination_nat {
+    name     = "testDNAT-1"
+    protocol = "TCP"
+    front_end_config {
+      public_ip_address_id = azurerm_public_ip.test.id
+      port                 = 8081
+    }
+    back_end_config {
```

Comment:
some of these resources uses `backend` and this one uses `back_end` - we should make these consistent as `backend`/`frontend`?